ad-first-campaign.py

```diff
@@ -0,0 +1,66 @@
+def main():
+
+    import sem
+    import math
+    import numpy as np
+    import os
+    import matplotlib
+    matplotlib.use('Agg')
+    import matplotlib.pyplot as plt
+    from collections import OrderedDict 
+
+    # Define campaign parameters
+    ############################
+
+    script = "evaluate_scheduler_qd_dense_scenario"
+    cmp_name = "basic-vs-cbaponly-v2"        # define a campaign name in order to create different corresponding to a specific configuration
```

Comment:
not that much longer but definitely more clear :)
```suggestion
    campaign_name = "basic-vs-cbaponly-v2"        # define a campaign name in order to create different corresponding to a specific configuration
```